model-optimizer/mo/ops/deformable_convolution.py

```diff
@@ -13,16 +13,18 @@ def __init__(self, graph: Graph, attrs: dict):
         super().__init__(graph, {
             'type': __class__.op,
             'op': __class__.op,
-            'version': 'opset1',
+            'version': 'opset8',
             'infer': Convolution.infer,
             'group': 1,
             'deformable_group': 1,
             'multiplication_transparent': True,
             'multiplication_transparent_ports': [(0, 0), (2, 0)],
```

Comment:
I can't find these two attributes in the specification. Please explain why they are needed in a comment.

Reply:
these attributes are marked as required in the spec, so we have to provide some values when extracting them from different frameworks, it can be different values for different frameworks. For required attributes we don't have default values.

Reply:
I'm not sure about these attributes, it is related to some preprocessing in MO, more precisely with BinarizeWeightsM1P1 transformation.

Reply:
I see your point. It makes sense to have it.